webapp/src/components/molecules/Login.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { Button } from 'primereact/button';
+import { Dropdown } from 'primereact/dropdown';
+import { Link } from 'react-router-dom';
+import 'primereact/resources/themes/saga-blue/theme.css';
```

Comment:
You'll need to `pnpm install primereact` (and maybe more) and commit both the `package.json` and `pnpm-lock.json` files to the repo. If you run `pnpm install` inside the `webapp` folder it _should_ remove anything that isn't in the current package.json if I recall correctly!

This probably works on your machine because you checked out someone else's branch that had those installed already so they stuck around when you switched back to this branch. However we need to ensure this branch/these commits work on their own :) 

--
Yup here I ran `pnpm install` on your branch and it seems I remove 5 depedencies (only showing some in the screenshot) so try running pnpm install then see what's missing
<img width="445" alt="image" src="https://user-images.githubusercontent.com/485809/149822061-5c9434d6-ead5-4a09-b3fe-082cfe45262d.png">